src/commands/default.ts

```diff
@@ -110,7 +122,30 @@ export default async function defaultMain(args: Argv) {
     if (args.push === true) {
       await execa("git", ["push", "--follow-tags"], { cwd });
     }
-    if (args.github !== false && config.repo?.provider === "github") {
+    if(args.pullRequest === true) {
+      // Create a new branch
+      await execa("git", ["checkout", "-b", "v" + config.newVersion], { cwd });
+      // Push the branch to the origin
+      await execa("git", ["push", "-u", "origin", "v" + config.newVersion], { cwd });
+    }
+    // Maybe we can create a new command to do this? (we don't want to create a release an a pull request at the same time but I don't know if this must be managed from the config or from the CLI)
```

Comment:
Yes, we can support `changelogen github pull-request|issue` (inside `commands/github.ts`)